fixtures/v2/api/listCharges/success.http

```diff
@@ -0,0 +1,14 @@
+HTTP/1.1 200 OK
+Date: Tue, 24 Oct 2023 09:52:55 GMT
+Connection: close
+X-RateLimit-Limit: 2400
+X-RateLimit-Remaining: 2397
+X-RateLimit-Reset: 1698143967
+Content-Type: application/json; charset=utf-8
+X-WORK-WITH-US: Love automation? So do we! https://dnsimple.com/jobs
+Cache-Control: no-store, must-revalidate, private, max-age=0
+X-Request-Id: a57a87c8-626a-4361-9fb8-b55ca9be8e5d
+X-Runtime: 0.060526
+Transfer-Encoding: chunked
```

Comment:
Is this correct? The body does not appear to match the [chunked encoding format](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Transfer-Encoding#chunked). This appears to break the Java tests. Apples to the other fixtures too.

Reply:
Fixtures have been updated. And indeed we do not use chunked encoding in our live environments.